scripts/index.js

```diff
@@ -0,0 +1,133 @@
+const state = {
+    tempCount: 70
+};
+
+const increaseTemp = () => {
+    state.tempCount += 1
+    const count = document.querySelector("#temperature");
+    count.textContent = state.tempCount;
+
+    if (state.tempCount > 79) {
+        red();
+        hot();
+    } else if (state.tempCount > 69) {
+        orange();
+        mild();
+    } else if (state.tempCount > 59)  {
+        yellow();
+        regular();
+    } else if (state.tempCount > 49) {
+        green();
+        cool();
+    } else {
+        blue();
+        cold();
+    };
+};
+
+const decreaseTemp = () => {
+    state.tempCount -= 1
+    const count = document.querySelector("#temperature");
+    count.textContent = state.tempCount;
+
+    if (state.tempCount > 79) {
+        red();
+        hot();
+    } else if (state.tempCount > 69) {
+        orange();
+        mild();
+    } else if (state.tempCount > 59)  {
+        yellow();
+        regular();
+    } else if (state.tempCount > 49) {
+        green();
+        cool();
+    } else {
+        blue();
+        cold();
+    };
+};
+
+const blue = () => {
+    const currentTemp = document.querySelector("#temperature");
+    currentTemp.className = `blue`;
+};
+const green = () => {
+    const currentTemp = document.querySelector("#temperature");
+    currentTemp.className = `green`;
+};  
+const yellow = () => {
+    const currentTemp = document.querySelector("#temperature");
+    currentTemp.className = `yellow`;
+};  
+const orange = () => {
+    const currentTemp = document.querySelector("#temperature");
+    currentTemp.className = `orange`;
+}; 
+const red = () => {
+    const currentTemp = document.querySelector("#temperature");
+    currentTemp.className = `red`;
+};
+
+const updateName = (event) => {
+    const cityName = document.querySelector("#sampleCity");
+    cityName.textContent = event.target.value;
+};
+
+const resetName = () => {
+    const cityName = document.querySelector("#sampleCity");
+    cityName.textContent = "Seattle"
+}
+
+const cold = () => {
```

Comment:
If we move to a more data-driven approach to updating the ground, we might not need these per-ground functions.